src/DataTable/tests/DataTable.test.jsx

```diff
@@ -184,86 +182,48 @@ describe('<DataTable />', () => {
     [{ manualSortBy: true, manualFilters: true, manualPagination: true, pageCount: 1 }, { manualFilters: true, manualPagination: true, manualSortBy: true }],
   ])('calls useTable with the correct manual settings %#', (additionalProps, expected) => {
     const spy = jest.spyOn(reactTable, 'useTable');
-    mount(<DataTableWrapper {...props} {...additionalProps} />);
+    render(<DataTableWrapper {...props} {...additionalProps} />);
     expect(spy.mock.calls[0][0].manualFilters).toEqual(expected.manualFilters);
     expect(spy.mock.calls[0][0].manualPagination).toEqual(expected.manualPagination);
     expect(spy.mock.calls[0][0].manualSortBy).toEqual(expected.manualSortBy);
   });
-  it('passes the initial state to useTable', () => {
+  test('passes the initial state to useTable', () => {
     const spy = jest.spyOn(reactTable, 'useTable');
     const initialState = { foo: 'bar' };
-    mount(<DataTableWrapper {...props} initialState={initialState} />);
+    render(<DataTableWrapper {...props} initialState={initialState} />);
     expect(spy.mock.calls[0][0].initialState).toEqual(initialState);
   });
-  it('displays loading state', () => {
-    const wrapper = mount(<DataTableWrapper {...props} isLoading />);
-    const tableContainer = wrapper.find('.pgn__data-table-container');
-    const spinner = wrapper.find('.pgn__data-table-spinner');
-    expect(tableContainer.hasClass('is-loading')).toEqual(true);
-    expect(spinner.exists()).toEqual(true);
-  });
+  test('displays loading state', () => {
+    const { container } = render(<DataTableWrapper {...props} isLoading />);
+    const tableContainer = container.querySelector('.pgn__data-table-container.is-loading');
```

Comment:
Maybe it will be better to remove this check since it uses implementation details? Below `spinner` should be found by `data-testid` instead of `querySelector`.